multisrc/overrides/madara/creepyscans/src/CreepyScans.kt

```diff
@@ -47,9 +56,37 @@ class CreepyScans : Madara(
 
     // Search
 
-    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
-        if (query.isNotBlank()) throw Exception("Search not available")
+    override fun fetchSearchManga(
+        page: Int,
+        query: String,
+        filters: FilterList,
+    ): Observable<MangasPage> {
+        return if (query.isNotBlank()) {
+            val form = FormBody.Builder()
+                .add("action", "wp-manga-search-manga")
+                .add("title", query)
+                .build()
+            client.newCall(POST("$baseUrl/wp-admin/admin-ajax.php", headers, form)).asObservableSuccess().map { res ->
+                json.parseToJsonElement(res.body.string()).jsonObject.let { obj ->
+                    if (obj["success"]!!.jsonPrimitive.content == "false") {
```

Comment:
```suggestion
                    if (obj["success"]!!.jsonPrimitive.boolean == false) {
```
https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-json/kotlinx.serialization.json/boolean.html

Reply:
Right, slipped my mind.